src/main/java/com/example/DataService/CurrencyService.java

```diff
@@ -39,14 +42,16 @@
 	@Value("${coin-api-url}")
 	private String uri;
 	
+	@Value("${coin-api-currency-exchange-url}")
```

Comment:
You could have used the same base url and just add different paths where needed - like this you have a lot of config properties which are almost the same